buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -483,16 +483,16 @@ public Stream<String> logLines() throws IOException {
     @Override
     public synchronized void start() {
         LOGGER.info("Starting `{}`", this);
-        if (System.getProperty("tests.opensearch.secure") != null
```

Comment:
As you identified the issue, could we not pass `tests.opensearch` props to the node instead of modifying properties? I am not sure what the impact this change would make (renaming properties) since we are basically modifying publicly available Gradle plugin

Reply:
> As you identified the issue, could we not pass tests.opensearch props to the node instead of modifying properties?

Yes, we could do this. The main reason I didnt do it was that the credentials did not seem to belong in the system properties for the node itself - in other words, the running node does not need these credentials for anything.

That being said, I was thinking that maybe just adding an option to set the `credentials` directly via `cluster.setCredentialsForHealthCheck` or `cluster.setCredentials` might make the most sense.

Reply:
Or @reta , alternatively, I see that there is a `user` option in the TestClusterConfiguration: https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/testclusters/TestClusterConfiguration.java#L125. 

In the OpenSearchNode, its a no-op: https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java#L706, but Im wondering if we can just put the following (which is what we do in the [start](https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java#L486-L496) with the sys props):
``` java
    @Override
    public void user(Map<String, String> userSpec) {
            this.credentials.get(0).put("username", userSpec.get("username")))
            this.credentials.get(0).put("password",  userSpec.get("password")));
    }
```

Im not really sure what outside of this health check credentials/user would be used for aside from. @cwperks do you think this sounds good?

Reply:
> Or @reta , alternatively, I see that there is a `user` option in the TestClusterConfiguration: https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/testclusters/TestClusterConfiguration.java#L125.

You mean as an alternative mechanism to system properties?

Reply:
Right, I updated the PR and confirmed it works for my k-NN change locally.